File: task3.py
def task3(start_number:int, end_number:int)->list:
    def is_prime(num):
        for div in range(2, int(num ** 0.5) + 1):
            if num % div == 0:
                return False
        return True
    return [number for number in range(start_number, end_number + 1) if number > 0 and is_prime(number)]

File: task3.js
function task3(startNumber, endNumber) {
    function isPrime(num) {
      for (let div = 2; div <= Math.sqrt(num); div++) {
        if (num % div === 0) {
          return false;
        }
      }
      return true;
    }
    return Array.from({ length: endNumber - startNumber + 1 }, (_, i) => startNumber + i).filter(number => number > 0 && isPrime(number));
  }